securing_apps/topics/oidc/oidc-overview.adoc

```diff
@@ -1,4 +1,4 @@
-== OpenID Connect
+== Configuring OpenID Connect client adapters
```

Comment:
Not sure if "client adapters" make sense. You are actually configuring clients. Adapters are the components that you rely on to configure/secure applications.

Perhaps "Securing applications using OpenID Connect" ?